### PR TITLE
Update comments in SendToHelix example

### PR DESCRIPTION
**Pull request by jonfortescue**

### CODE REVIEW

File: Documentation/AzureDevOps/SendingJobsToHelix.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ The list of available Helix queues can be found on the [Helix homepage](https://
       HelixType: type/tests
       # HelixBuild: $(Build.BuildNumber) -- This property is set by default
       HelixTargetQueues: Windows.10.Amd64.Open;Windows.81.Amd64.Open;Windows.7.Amd64.Open # specify appropriate queues here; see https://helix.dot.net/ for a list of queues
-      HelixAccessToken: $(BotAccount-dotnet-github-anon-kaonashi-bot-helix-token) # this token is only for external (public) builds
+      # HelixAccessToken: $(HelixAccessToken) # this token is only for internal builds
```
Comment (riarenas) [inline Documentation/AzureDevOps/SendingJobsToHelix.md]:
```suggestion
      # HelixAccessToken: $(HelixAccessToken) this token is only for internal builds
```

Comment (riarenas) [reply Documentation/AzureDevOps/SendingJobsToHelix.md]: Should we include that in order to get that variable, you should reference the Dotnet-HelixApi-Access variable group?

Comment (jonfortescue) [reply Documentation/AzureDevOps/SendingJobsToHelix.md]: It's mentioned a couple of paragraphs up.

Comment (riarenas) [reply Documentation/AzureDevOps/SendingJobsToHelix.md]: I didn't read far up enough :(

Comment (jonfortescue) [reply Documentation/AzureDevOps/SendingJobsToHelix.md]: it was easy to miss since I didn't edit that part! haha

**Pull request merged**